<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-08T22:16:49 IST" name="Suite" finished-at="2025-02-08T22:16:52 IST" duration-ms="2898">
    <groups>
    </groups>
    <test started-at="2025-02-08T22:16:49 IST" name="Test" finished-at="2025-02-08T22:16:52 IST" duration-ms="2898">
      <class name="com.selenium.tests.EndToEndTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.selenium.tests.EndToEndTest@36ebc363]" started-at="2025-02-08T22:16:46 IST" name="setUp" finished-at="2025-02-08T22:16:49 IST" duration-ms="3320" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="endToEndTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.selenium.tests.EndToEndTest@36ebc363]" started-at="2025-02-08T22:16:50 IST" name="endToEndTest" data-provider="loginData" finished-at="2025-02-08T22:16:52 IST" duration-ms="1791" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[John]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Doe]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at com.selenium.tests.EndToEndTest.endToEndTest(EndToEndTest.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endToEndTest -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.selenium.tests.EndToEndTest@36ebc363]" started-at="2025-02-08T22:16:52 IST" name="afterMethod" finished-at="2025-02-08T22:16:52 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=endToEndTest status=FAILURE method=EndToEndTest.endToEndTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.selenium.tests.EndToEndTest@36ebc363] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.selenium.tests.EndToEndTest@36ebc363]" started-at="2025-02-08T22:16:52 IST" name="tearDown" finished-at="2025-02-08T22:16:52 IST" duration-ms="157" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.selenium.tests.EndToEndTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
